README.md

```diff
@@ -1,14 +1,77 @@
-# Create Blazor ListView In Blazor Webassembly App
-A quick start Blazor WebAssembly app that allows you to configure the Blazor ListView component from Syncfusion with its available options. This project contains simple code customizations, as well as some important features like how to bind different types of data, group list items, add checkboxes to a list, and add nested list items.
+ # Create Blazor ListView In Blazor Webassembly App
 
-Documentation: https://blazor.syncfusion.com/documentation/listview/getting-started/
-
-Online examples: https://blazor.syncfusion.com/demos/listview/default-functionalities
+The [Blazor ListView](https://www.syncfusion.com/blazor-components/blazor-listview?utm_source=github&utm_medium=listing&utm_campaign=blazor-lists-github-samples) is a list-like interface that allows you to select an item or multiple items. A quick start Blazor WebAssembly app that allows you to configure the Blazor ListView component from Syncfusion with its available options. This project contains simple code customizations, as well as some important features like how to bind different types of data, group list items, add checkboxes to a list, and add nested list items.

```

Comment:
UTM param should be 

?utm_source=github&utm_medium=listing&utm_campaign=blazor-listview-github-samples

Reply:
resolved